src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -37,11 +37,13 @@ export type JobStatus =
 export type BlastSubmission = {
   submittedData: {
     species: Species[];
-    sequences: string[]; // Or perhaps parsed sequences?
+    sequences: { id: number; value: string }[]; // TODO: consider whether to have strings or parsed sequences
```

Comment:
What do you mean by `parsed sequences`?

Reply:
https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/tools/blast/types/parsedInputSequence.ts

Reply:
I think `string[]` is fine unless we convert it to `parsed sequence` later on demand, multiple times.

Reply:
As discussed, storing it as parsed sequence will be better but it can be done in a different PR.
I guess we can retain this todo comment just so we don't miss it.